docs/node/mainnet/README.md

```diff
@@ -105,7 +104,7 @@ Feel free to use other IAS proxies besides the one provided here or [run your ow
 ### Key Manager
 
 * Oasis Core version:
-  * [22.2.11](https://github.com/oasisprotocol/oasis-core/releases/tag/v22.2.11)
+  * [23.0.7](https://github.com/oasisprotocol/oasis-core/releases/tag/v23.0.7)
```

Comment:
These are not current parameters, but will be after we upgrade everything.

Reply:
This is a draft, to be merged after the upgrade.